README.md

```diff
@@ -137,8 +137,8 @@ anywhere.
 In other words, kube-state-metrics itself is designed to be another source for
 Heapster (although this is not currently the case).
 
-Additionally, some monitoring systems such as Prometheus do not use Heapster
-for metric collection at all and instead implement their own. Having
+Additionally, some monitoring systems such as Prometheus can use both Heapster
+and kube-state-metrics as scrape target for metric collection. Having
```

Comment:
Yes, but as far as I understand that does not expose the state of the metrics that heapster itself collects, but rather only it's own state, that Prometheus could alert on.